### ISSUE TITLE
some additional info for people who didnt watch complete video

### ISSUE THREAD

**Issue by drascom** [johnkomarnicki/FireBlogs-YouTube#8]
hi john, thank you for great tutorial and very detailed explanation about vue and firebase.but better to add some info about npm install-g firebase-tools and firebase init and select their own firebase application steps to the project main page at installation section.
the people who is didnt watched all video "like me" will getting error while try to run project 

thanks...

**Comment by m-schou** [comment]
Yes please update it so it runs correctly right away. Would help alot.
